bagging.md

```diff
@@ -1,22 +1,32 @@
 # Baggers
 
 - Getting set up as a Bagger
-  - Talk to your DataRescue guide to make sure you can become a Bagger.
+  - Apply to become a Bagger 
+    - By asking your DataRescue guide or by filling out [this form](XXX)
```

Comment:
Good point. I hadn't thought of the idea of different forms for each event. I was just thinking that event guide leaders could get access to approve people in the main form at the event. But, what do you think is a better way of handling this?